src/interface.cc

```diff
@@ -97,47 +97,47 @@ NLV_WORKER_EXECUTE(Interface, Undefine)
 NLV_LOOKUP_BY_VALUE_EXECUTE_IMPL(Interface, LookupByName, virInterfaceLookupByName)
 NAN_METHOD(Interface::LookupByName)
 {
-  NanScope();
-  if (args.Length() < 2 ||
-      (!args[0]->IsString() && !args[1]->IsFunction())) {
-    NanThrowTypeError("You must specify a valid Interface name and callback.");
-    NanReturnUndefined();
+  Nan::HandleScope scope;
+  if (info.Length() < 2 ||
+      (!info[0]->IsString() && !info[1]->IsFunction())) {
+    Nan::ThrowTypeError("You must specify a valid Interface name and callback.");
+    return;
   }
 
-  Local<Object> object = args.This();
-  if (!NanHasInstance(Hypervisor::constructor_template, object)) {
-    NanThrowTypeError("You must specify a Hypervisor instance");
-    NanReturnUndefined();
+  Local<Object> object = info.This();
+  if (!Nan::New(Hypervisor::constructor_template)->HasInstance(object)) {
```

Comment:
Okay, thus concludes my review-without-actuallty-compiling :smile: I think we're good to go here. I'm going to merge but hold off on an npm publish as to give others a chance to test